ui/dashboardApp/layout/main/Sider/SiderService.ts

```diff
@@ -0,0 +1,36 @@
+import { useState } from 'react'
```

Comment:
I'm not sure whether it is fine to implement this as a service, as Service is usually considered something stateful and long-running, while the implementation here is stateless and is a react-hook only thing.

Reply:
This is a borrowed concept. https://angular.io/guide/architecture-services
This is kind of concept `Business Service`, rather than concept `Technical Service`? Which means kind of business logic that can be extracted, standalone, easy for test.

Reply:
What's meaning about "stateless and is a react-hook only here". You mean no side effect?
Maybe stateless service also make sense?